app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/TimeUtils.scala

```diff
@@ -99,6 +100,8 @@ object TimeUtils {
                    .flatMap(date =>
                      if (maximumDateInclusive.exists(_.isBefore(date)))
                        Left(FormError(dateKey, "error.tooFarInFuture"))
+                     else if (date.isBefore(minimumDate))
```

Comment:
this case should go below the `if (minimumDateInclusive.exists(_.isAfter(date)))` case